test/tools/jsontests/BlockChainTests.cpp

```diff
@@ -407,7 +413,9 @@ void doBlockchainTests(json_spirit::mValue& _v, bool _fillin)
 			ImportTest::compareStates(postState, blockchain.topBlock().state());
 		}
 	}//for tests
-	TestOutputHelper::finishTest();
+
+	if (!skipTestOutput)
+		TestOutputHelper::finishTest();
```

Comment:
Previously This was executed when `Options::get().fillchain == flase`, now it's not, is this correct?